pkg/package.json

```diff
@@ -4,7 +4,7 @@
     "John Sahhar <john@entropy.xyz>"
   ],
   "description": "x25519 & chacha20poly1305",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
https://semver.org/

dumb question: would this be a patch?

Reply:
Yeah i wasn't sure, i think its sort of grey area.  In some cases `encrypt_and_sign` will give different output than before, because it used to give a uint8array containing the word 'failed' on failure, now it throws an error.